### PR TITLE
Remove erroneous and missing nerdfont icons

### PR DESCRIPTION
**Pull request by romulof**

### CODE REVIEW

**Comment by dritter** [comment]
Hi @romulof !

Thanks for your contribution. To be honest, I am a bit puzzled about this change. I see that you removed a bunch of icons here, especially for icons that are in Nerdfonts complete (like `VCS_GIT_ICON`; `uF113`).


So these code point work for NerdFonts only in the `complete` variant. Do you use such a variant, or did you patch the font yourself?

**Comment by romulof** [comment]
This should be a GIT icon, not that octocat thing from Github :)

I’m using the last release of the Hasklig font, complete set.

File: functions/icons.zsh [hunk 1]
```diff
@@ -213,22 +213,22 @@ case $POWERLEVEL9K_MODE in
       VCS_COMMIT_ICON                $'\uE729 '             # 
       VCS_BRANCH_ICON                $'\uF126 '             # 
       VCS_REMOTE_BRANCH_ICON         $'\uE728 '             # 
-      VCS_GIT_ICON                   $'\uF113 '             # 
+      VCS_GIT_ICON                   $''                    # 
```
Comment (dritter) [inline functions/icons.zsh]: Btw. the official GIT icon is in NerdFonts:
bildschirmfoto 2018-04-21 um 15 37 46

File: functions/icons.zsh [hunk 2]
```diff
@@ -213,22 +213,22 @@ case $POWERLEVEL9K_MODE in
...
       VCS_GIT_GITHUB_ICON            $'\uE709 '             # 
       VCS_GIT_BITBUCKET_ICON         $'\uE703 '             # 
       VCS_GIT_GITLAB_ICON            $'\uF296 '             # 
-      VCS_HG_ICON                    $'\uF0C3 '             # 
-      VCS_SVN_ICON                   $'\uE72D '             # 
```
Comment (dritter) [inline functions/icons.zsh]: The SVN Icon seems like we had previously a not-so-great choice here:
bildschirmfoto 2018-04-21 um 15 35 26

File: functions/icons.zsh [hunk 3]
```diff
@@ -213,22 +213,22 @@ case $POWERLEVEL9K_MODE in
...
+      VCS_HG_ICON                    $''                    # 
```
Comment (dritter) [inline functions/icons.zsh]: Why was the HG Icon removed? Okay, it is not the official Logo, but its better than nothing.
bildschirmfoto 2018-04-21 um 15 36 24

**Pull request closed without merging**